packages/plugins/ScamWarning/src/SNSAdaptor/index.tsx

```diff
@@ -33,7 +33,7 @@ const sns: Plugin.SNSAdaptor.Definition = {
         const links = useMemo(() => {
             const x = extractTextFromTypedMessage(props.message)
             if (x.none) return null
-            return parseURLs(x.val)
+            return parseURLs(x.val, false)
```

Comment:
why you're going to use an invalid URL?

Reply:
I hope get the `a.com` from message.

Reply:
Maybe you could create another one: `parseLinks`.